src/iotjs_util.c

```diff
@@ -115,7 +115,8 @@ void iotjs_buffer_release(char* buffer) {
 }
 
 void print_stacktrace(void) {
-#if defined(__linux__) && defined(DEBUG) && !defined(__OPENWRT__)
+#if !defined(NDEBUG) && defined(__linux__) && defined(DEBUG) && \
```

Comment:
How is this related to the windows change?

Reply:
There were a code duplication in `napi_fatal_error`. I changed it to use this function, but this change is needed to avoid linker error in release builds.